patches/chrome-browser-importer-external_process_importer_client.cc.patch

```diff
@@ -0,0 +1,22 @@
+diff --git a/chrome/browser/importer/external_process_importer_client.cc b/chrome/browser/importer/external_process_importer_client.cc
+index 429c16dc6a56fcd0ea1d674508c1927a387e1905..0f29ba75ffce20b1104e36b4ba1aa78fd28fc50a 100644
+--- a/chrome/browser/importer/external_process_importer_client.cc
++++ b/chrome/browser/importer/external_process_importer_client.cc
+@@ -8,6 +8,7 @@
+ 
+ #include "base/bind.h"
+ #include "base/strings/string_number_conversions.h"
++#include "brave/grit/generated_resources.h"
```

Comment:
Please rebase to master. We don't need to patch this now, 
see https://github.com/brave/brave-core/pull/179/commits/5b4838a4f32f7b38f5420287b0f547f4514aba8f